helm/examples/azure/hpcc-azurefile/values.schema.json

```diff
@@ -71,6 +71,14 @@
         "shareName": {
           "description": "Azure storage share name",
           "type": "string"
+        },
+        "secretName": {
+          "description": "the secretName, the storage account, for current plane",
```

Comment:
I think better reworded to be like the common description this overrides, e.g.:

"Kubernetes secret name to use for the Azure storage account for this plane"

Reply:
Fixed